tests/test_name.py

```diff
@@ -155,9 +160,6 @@ def testCompare3(self):
     def testCompare4(self):
         self.failUnless(dns.name.root != 1)
 
-    def testCompare5(self):
-        self.failUnless(dns.name.root < 1 or dns.name.root > 1)
```

Comment:
This test has been removed because it relies on type comparison which has been removed in python3. See http://eli.thegreenplace.net/2016/comparing-types-in-python-3/ which explains this very well.